types/v56/types.go

```diff
@@ -2308,6 +2308,17 @@ type VimObjectRef struct {
 	VimObjectType string     `xml:"VimObjectType"`
 }
 
+// Type: VimObjectRefType
+// Namespace: http://www.vmware.com/vcloud/extension/v1.5
+// https://vdc-repo.vmware.com/vmwb-repository/dcr-public/7a028e78-bd37-4a6a-8298-9c26c7eeb9aa/09142237-dd46-4dee-8326-e07212fb63a8/doc/doc/types/VimObjectRefsType.html
+// Description: Represents the Managed Object Reference (MoRef) and the type of a vSphere object.
+// Since: 0.9
+type VimObjectRefCreate struct {
```

Comment:
Could become private now.

Reply:
Then I need to move it to another go file

Reply:
moved